question7/demo.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shopping List</title>
+</head>
+<body>
+    <h1>My Shopping List</h1>
+    <form>
+        <label>Enter a new item:</label>
+        <input type="text" id="item-input" placeholder="Enter a new item" required>
+        <button id="add-button">Add Item</button>
+        <ul id="shopping-list"></ul>
+    </form>
+
+    <script>
+        const addButton = document.getElementById('add-button');
+        const itemInput = document.getElementById('item-input');
+        const shoppingList = document.getElementById('shopping-list');
+
+        addButton.addEventListener('click', () => {
```

Comment:
this will work but we can call the method froom template itself using onclick right?